src/main/java/me/karakelley/http/responses/Unauthorized.java

```diff
@@ -0,0 +1,19 @@
+package me.karakelley.http.responses;
+
+import me.karakelley.http.Response;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Unauthorized extends Response {
+
+  public String getStatus() {
+    return "401 Unauthorized";
+  }
+
+  public Map<String, String> getHeaders() {
+    Map<String, String> headers = new HashMap<>();
+    headers.put("WWW-Authenticate", "Basic realm=My Server");
```

Comment:
Nice job consistently getting your responses up to spec. You've been nailing the response codes, and now this is one more thing that could easily be forgotten.